### ISSUE TITLE
Allow admins to generate password reset links

### ISSUE THREAD

**Issue by Altair-Bueno** [Altair-Bueno/ubay#9]
### How does the admin generate a password reset link

1. Admin clicks a link
2. Ubay inserts the tuple username & random string (length 20)
4. Ubay generates the link
5. Ubay sends the link to the admin
6. Admin sends the reset link to the user

### How does a user reset his password?

By following a url like this:

```url
/auth/resetPassword?username=<username>&passwordChangeID=<changeID>
```

Note: The username and changeID relates to both columns of the table `password_reset`

Note2: A password reset can only occur once with the same link

**Comment by Altair-Bueno** [comment]
Important: Add this page to the filter mapping on web.xml

**Comment by Altair-Bueno** [comment]
Closed on 68af36943081ee3374a09ab98b09733f6c8babc8
